ci/is-container-log-line-present.sh

```diff
@@ -9,18 +9,13 @@ main () {
     local container_name_substring=$1; shift
     local log_message=$1; shift
 
-    local container_hash=$(docker ps --filter "name=$container_name_substring" --quiet | head -1)
+    local container_hash=$(docker ps --filter "name=$container_name_substring" --quiet)
```

Comment:
Not sure why `| head -1` didn't make it here.

Reply:
It should fail when there are multiple containers matching the filter